### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by Venky-2107**

### CODE REVIEW

File: backend/src/controllers/adminLoginController.js [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
 
 const loginController = {};
 
 loginController.registerUser = async function (req, res) {
   try {
-    let newUser = new admin_login({ _id: uuid.v4(), ...req.body });
-    newUser.save();
+    let { password, secret } = req.body;
+    let hashedPassword = await bcrypt.hash(password, 12);
+    console.log("salt", await bcrypt.genSalt(10));
```
Comment (TejoVarma) [inline backend/src/controllers/adminLoginController.js]: comment the line

File: backend/src/controllers/adminLoginController.js [hunk 2]
```diff
@@ -24,16 +34,41 @@ loginController.getAdminUser = async (req, res) => {
...
+        status: "failed",
+        message: "user not found",
+      });
+    }
+  } catch (err) {
+    res.status(400).send({ status: "failed" });
+  }
+};
+
 loginController.updateUserById = async (req, res) => {
```
Comment (TejoVarma) [inline backend/src/controllers/adminLoginController.js:67]: naming conventions should be changed as the login controllers doesnt need to update

File: backend/src/controllers/adminLoginController.js [hunk 3]
```diff
@@ -46,7 +81,7 @@ loginController.updateUserById = async (req, res) => {
 
 loginController.deleteUserById = async (req, res) => {
```
Comment (TejoVarma) [inline backend/src/controllers/adminLoginController.js:80]: same as edit

File: backend/src/controllers/adminLoginController.js [hunk 4]
```diff
@@ -55,7 +90,7 @@ loginController.deleteUserById = async (req, res) => {
 
 loginController.deleteUsers = async (req, res) => {
```
Comment (TejoVarma) [inline backend/src/controllers/adminLoginController.js:89]: same as above

**Review by TejoVarma — CHANGES_REQUESTED** [review]

File: backend/src/controllers/adminLoginController.js (hunk 1 above)
Comment (Venky-2107) [reply backend/src/controllers/adminLoginController.js]: removed console.log

**Review by TejoVarma — APPROVED** [review]

**Pull request merged**